web/impact/impact/graphql/types/expert_profile_type.py

```diff
@@ -97,6 +102,30 @@ def resolve_current_mentees(self, info, **kwargs):
     def resolve_previous_mentees(self, info, **kwargs):
         return _get_mentees(self.user, ENDED_PROGRAM_STATUS)
 
+    """
+    fetch a program role assigned to an user and those assigned to the startups
+    the user belong to
+
+    Time, Space, Query Complexity
+    Query: amount to the query complexity of the two helper function that
+    access the DB (see functions for query comp analysis for each)
+
+    Time/Space amount to time and space complexity of the three helper functions
+    (see functions for time/space comp analysis for each)
+    """
+    def resolve_program_roles(self, info, **kwargs):
```

Comment:
seems like this is a duplicate method. can we extract this out to a shared function?

Reply:
@shanbady what other function does this duplicate?

Reply:
resolve_program_roles shows up here:

https://github.com/masschallenge/impact-api/pull/378/files/9f26df4253eadcf28a4c6d2f87547a36025ba953#diff-d0b1a8955f0c1743915a8a2c97703158R63
and here:
https://github.com/masschallenge/impact-api/pull/378/files/9f26df4253eadcf28a4c6d2f87547a36025ba953#diff-2807098ef5f50df497dcaf6d1591b111R116

Reply:
@PrinceDavis This needs to resolved

Reply:
This has been resolved

Reply:
Thanks for the clarification

Reply:
FWIW, code climate thinks this hasn't been resolved (still showing a duplication warning)